pkg/apis/camel/v1/trait/mount.go

```diff
@@ -33,7 +33,9 @@ type MountTrait struct {
 	// Syntax: [configmap|secret]:name[/key][@path], where name represents the resource name, key optionally represents the resource key to be filtered and path represents the destination path
 	Resources []string `property:"resources" json:"resources,omitempty"`
 	// A list of Persistent Volume Claims to be mounted. Syntax: [pvcname:/container/path]
-	Volumes []string `property:"volumes" json:"volumes,omitempty"`
+	PersistentVolumeClaims []string `property:"persistent-volume-claims" json:"persistentVolumeClaims,omitempty"`
```

Comment:
We cannot change this without a period of deprecation notice. This change affects the API specification, so, it must be reverted. It was an unfortunate choice when we created it as it really was meant to be PersistentVolumeClaims. We may think to change this when moving to a major API upgrade.